### DISCUSSION TITLE
Feature Request: Add toggle to always show specific bed point data

### DISCUSSION THREAD

**Discussion by MuttCollective** [jneilliii/OctoPrint-BedLevelVisualizer#506] [Ideas]
First off, thank you for the development of this plugin and so many others. Huge fan and love this plugin. 

I've been meaning to ask for a while if you can add a toggle option to always show the pop-up info for at least the 4 corners of the bed? Instead of hovering it would be nice to just have the results at a glance. I know I can turn on the data table and have it below the graph, however, the graph and table data aren't always relative in position so it makes more of a study than a quick check. 

Here is a mock up to help explain. 
![Bed Level Visualizer](https://user-images.githubusercontent.com/89160173/149242446-49d7bb7d-41f6-48fb-a5c9-4aabd5d3d979.jpg)

Thanks!

**Comment by jneilliii** [comment]
Added as feature request [here](https://github.com/jneilliii/OctoPrint-BedLevelVisualizer/issues/508). I'll check into plotly to see how to achieve that. I know it's possible to display the markers, just not sure if it's all or nothing, or if I would have to implement the marker logic myself.
